package.json

```diff
@@ -31,9 +31,9 @@
     "react-dom": "16.8.3",
     "react-redux": "5.1.1",
     "react-router-dom": "4.3.1",
-    "react-scripts-ts": "3.1.0",
+    "react-scripts-ts": "2.17.0",
```

Comment:
What's the reasoning for downgrading react-scripts?

Reply:
Had to downgrade that to get the tests to run due to some bug in in `react-scripts-ts` v3. That package is the one we're trying to get away from because it's no longer maintained and `react-scripts` from facebook now supports TS. Will be done in a separate PR.